<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Renji</class>
 <widget class="QWidget" name="Renji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="toolButton_over">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="icon">
    <iconset resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">
     <normaloff>:/picture/exit.png</normaloff>:/picture/exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_back">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="icon">
    <iconset resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">
     <normaloff>:/picture/goback.png</normaloff>:/picture/goback.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonQuit">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>380</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">
     <normaloff>:/picture/end.png</normaloff>:/picture/end.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhite">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">:/picture/baizi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberBlack">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberWhite">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>180</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlack">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">:/picture/heizi.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonstart">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="C:/Users/28903/Desktop/h_b_qi/res.qrc">
     <normaloff>:/picture/start.png</normaloff>:/picture/start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/28903/Desktop/h_b_qi/res.qrc"/>
 </resources>
 <connections/>
</ui>
